src/main/java/com/netgrif/application/engine/configuration/CachingConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.netgrif.application.engine.configuration;
+
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableCaching
+public class CachingConfiguration {
+    @Bean
+    public CacheManager cacheManager() {
+        return new ConcurrentMapCacheManager("netsId", "netsIdentifier", "netsNewest", "netsCache");
```

Comment:
Is it possible to have cache names as Spring app properties and get them as a list? If a third-party project uses caching it would be better if they did not have to know these names and it would be easier to add custom caches. Also it would be better to have a single source of these names - no hardcoded strings in PetriNetService and CachingConfiguration.

Reply:
done